src/System/IO/UTF8.hs

```diff
@@ -14,6 +18,12 @@ import qualified Data.Text.Encoding as TE
 fixCRLF :: BS.ByteString -> BS.ByteString
 fixCRLF = BSL.toStrict . BSS.replace "\r\n" ("\n" :: BS.ByteString)
 
+readUTF8FilesTUnique :: [FilePath] -> IO [(FilePath, Text)]
+readUTF8FilesTUnique =  readUTF8FilesT . nubOrd
+
+readUTF8FilesT :: [FilePath] -> IO [(FilePath, Text)]
```

Comment:
Are we likely to want to use the "non-unique" version anywhere? I can't think of a place we might want to do that so I'd suggest only providing the unique version (and getting rid of the "Unique" suffix).

Reply:
Sure thing!